sdk/textanalytics/test-resources.json

```diff
@@ -35,18 +35,13 @@
         },
         "location": {
             "type": "string",
-            "defaultValue": "[resourceGroup().location]",
+            "defaultValue": "eastus",
```

Comment:
could this be controlled using a parameter in `tests.yml` instead? similar to [this](https://github.com/Azure/azure-sdk-for-js/blob/ceb5483caf27456c05c6ecb11ca5dcbd9ce339aa/sdk/cognitivelanguage/ai-language-text/tests.yml#L1-L5)

Reply:
Awesome. Thanks!